benches/rust/forth.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(test)]
+extern crate test;
+use test::Bencher;
+use forth::*;
+
+#[bench]
+fn simple_bench(b: &mut Bencher) {
+    let mut f = Forth::new();
+    b.iter(|| {
+        let x = f.eval("2 2 + : addtwo 2 + ; 2 addtwo drop drop 2 2 + : addone 1 + ; 2 addone addtwo drop drop");
+        if !x.is_ok() {panic!("bad");}
```

Comment:
```suggestion
        f.eval(
            "2 2 + : addtwo 2 + ; 2 addtwo drop drop 2 2 + \
            : addone 1 + ; 2 addone addtwo drop drop",
        )
        .unwrap();
```

`unwrap` causes a panic already, so we can use it here. If you still wish to include a more verbose panic message, use `expect`. The string has been split in two lines for formatting.